Emby.Server.Implementations/HttpServer/Security/AuthorizationContext.cs

```diff
@@ -263,29 +263,60 @@ private AuthorizationInfo GetAuthorization(HttpContext httpReq)
                 return null;
             }
 
-            // Remove uptil the first space
+            // Remove up until the first space
             authorizationHeader = parts[1];
-            parts = authorizationHeader.Split(',');
-
-            var result = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            return GetParts(authorizationHeader);
+        }
 
-            foreach (var item in parts)
+        /// <summary>
+        /// Get the authorization header components.
+        /// </summary>
+        /// <param name="authtorizationHeader">The authorization header.</param>
+        /// <returns>string</returns>
+        public static Dictionary<string, string> GetParts(string authtorizationHeader)
+        {
+            var result = new Dictionary<string, string>();
+            var escapeChars = new[] { '"', ',' };
```

Comment:
Don't allocate an array here, check both manually at line 288

Reply:
Right. I'll commit the change.